packages/manager/modules/dbaas-logs/src/logs/logs-constants.js

```diff
@@ -13,7 +14,7 @@ export default {
   SUCCESS: 'SUCCESS',
   suffixPattern: '^[a-z0-9_-]+$',
   ORDER_URL:
-    "/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs))",
+    "/order/express/#/new/express/resume?products=~(~(planCode~'logs-account~productId~'logs~region~'gra))",
```

Comment:
region should be dynamic right ?

Reply:
i'm not using this constant

Reply:
if it's not used maybe we can remove it ?

Reply:
done :)